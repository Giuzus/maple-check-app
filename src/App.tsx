import "./App.css";
import { createBrowserRouter, RouterProvider } from "react-router-dom";
import Home from "./pages/home/home.page.tsx";


const router = createBrowserRouter([
  { path: "/", element: <Home /> },
]);

function App() {
  return (
    <RouterProvider router={router} />
  );
}

export default App;
