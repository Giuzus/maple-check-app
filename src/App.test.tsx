import { describe, expect, test } from "vitest";
import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  test("should match snapshot", () => {
    //Arrange
    const page = render(<App />);

    //Assert
    expect(page.baseElement).toMatchSnapshot();
  });
});
