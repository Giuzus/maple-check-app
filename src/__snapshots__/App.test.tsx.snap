// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should match snapshot 1`] = `
<body>
  <div>
    <h1>
      /home
    </h1>
  </div>
</body>
`;
